package.json

```diff
@@ -12,7 +12,7 @@
     "lint-es": "eslint --cache --cache-location '.eslintcache/' --ext .js,.jsx,.ts,.tsx --max-warnings=0 src",
     "format-check": "node prettier.js lint",
     "format": "node prettier.js write",
-    "start": "razzle start  --inspect --inspect-port 9230",
+    "start": "NODE_OPTIONS='--max-old-space-size=2048' razzle start --inspect --inspect-port 9230",
```

Comment:
Hva gjør denne?

Reply:
Gir meir minne til prosessen. For å unngå memory heap exception.